configurationPanel.js

```diff
@@ -0,0 +1,78 @@
+class ConfigurationPanel extends Panel
+{
+constructor(application,db)
```

Comment:
nie potrzebujesz przekazywać osobno db do paneli, bo masz to dostępne poprzez `app.db`. Poza tym zrobiłeś to na tyle niespójnie, ze niektóre panele odbierają db, a niektóre nie